7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -178,14 +178,21 @@ map_query <- function(out_file, lake_cell_tile_xwalk, query_tiles, query_cells,
   grid_tiles <- grid_tiles %>%
     filter(tile_no %in% query_tiles)
 
+  tile_labels <- grid_tiles %>%
+    mutate(bbox=split(.,1:nrow(grid_tiles)) %>% purrr::map(sf::st_bbox)) %>%
```

Comment:
I think you can use `tile_no` instead of `1:nrow(grid_tiles)` (that's the suggestion part of this comment). I've not seen `split()` + `purrr::map()` called _inside_ a mutate before. That is pretty nifty. I probably would have solved this by doing a `group_by()` + `mutate()` but the list part is necessary so I like having the `purrr::map()` call.